site/docs/configuration.md

```diff
@@ -41,6 +41,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.compression-codec    | gzip               | Parquet compression codec                          |
 | write.parquet.compression-level    | null               | Parquet compression level                          |
 | write.avro.compression-codec       | gzip               | Avro compression codec                             |
+| write.orc.stripe-size-bytes | 67108864(64 MB) | Define the default ORC stripe size, in bytes. |
```

Comment:
nit: align `|` for each column unless it's too long.

Reply:
done